lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1775,8 +1778,9 @@ def api_client
         authentication = Google::Auth.get_application_default
         creds = GRPC::Core::CallCredentials.new(authentication.updater_proc)
         creds = ssl_creds.compose(creds)
-        @client = Google::Logging::V2::LoggingServiceV2::Stub.new(
-          'logging.googleapis.com', creds)
+        channel = GRPC::Core::Channel.new('logging.googleapis.com', nil, creds)
+        @client = Google::Cloud::Logging::V2::LoggingServiceV2Client.new(
+          channel: channel)
```

Comment:
[Optional] You can probably inline the `channel` expression in here.

Reply:
Changed.